src/translations/client.csv

```diff
@@ -800,6 +801,12 @@ event.action.declare.form.section.person.field.age.error,Error message for inval
 event.action.notAvailableForEvent,Shown when user tries to perform an action that is not available for the event,The action you're trying to perform is not available for this event anymore,L'action que vous essayez d'effectuer n'est plus disponible pour cet événement
 event.action.outbox-retry.label,Label for retry button for outbox,Retry,Réessayer
 event.action.review-correction.label,Label for review correction button in dropdown menu,Review correction request,Revoir la demande de correction
+event.edit.registerWithEdits.label,Label for "Register with edits" in edit action menu,Register with edits,Enregistrer avec modifications
+event.edit.declareWithEdits.label,Label for "Declare with edits" in edit action menu,Declare with edits,Déclarer avec modifications
+event.edit.registerWithEdits.description,Description for "Register with edits" in edit action menu,Are you sure you want to register this event with these edits?,Êtes-vous sûr de vouloir enregistrer cet événement avec ces modifications ?
+event.edit.declareWithEdits.description,Description for "Declare with edits" in edit action menu,Are you sure you want to edit this declaration? By confirming you are redeclaring this event and override past changes...,Êtes-vous sûr de vouloir modifier cette déclaration ? En confirmant, vous redéclarez cet événement et remplacez les modifications précédentes...
+event.edit.comment.label,The label for the comment,Comments,Commentaires
+editPageBanner.message,The message for the edit page banner,You are editing a record declared by {name} ({role} at {location}),Vous modifiez un dossier déclaré par {name} ({role} à {location})
```

Comment:
maybe run the pre-commit correctly before merging, to have the translations sorted